storage-access.bs

```diff
@@ -162,7 +164,10 @@ When invoked on {{Document}} |doc|, the <dfn export method for=Document><code>ha
 
     ISSUE: "same authority" here is a placeholder for a future concept that allows user agents to perform [=same site=] checks while adhering to additional security aspects such as the presence of a cross-site parent document, see [whatwg/storage#142](https://github.com/whatwg/storage/issues/142#issuecomment-1122147159). In practice, this might involve comparing the [=site for cookies=] or performing a [=same site=] check with the top-level document.
 
-1. [=Queue a global task=] on the [=permissions task source=] given |global| to [=/resolve=] |p| with |global|'s [=environment/has storage access=].
+1. If |global|'s [=environment/has storage access=] is true, [=/resolve=] |p| with true and return |p|.
```

Comment:
We should continue to queue a task for this case. But also, this seems wrong. You can have storage access but no cookie access, right? E.g., for top-level documents that seems possible.

Reply:
> You can have storage access but no cookie access, right? E.g., for top-level documents that seems possible.

Thanks for the review, Anne! To make sure I understand correctly, could you help clarify what you mean with this comment? Do you mean 
- "the global's `has storage access` bit could be true but the user could have changed settings so that cookies are not really accessible", or 
- "the document could have dom storage access but not cookie access "?

Reply:
The first of those. I think that could also happen for a cross-site document, if you had previously given storage access, but changed settings since then to block cookies.